angular/scripts/sync.sh

```diff
@@ -0,0 +1,5 @@
+# Copy core dist
```

Comment:
@liamdebeasi FYI - this could probably be propagated to the other `sync.sh` scripts. I only added one here as an example in case you didn't want to do it (I didn't want to spam you with the same suggestion 🤣)

Reply:
Thx! Fixed in https://github.com/ionic-team/ionic-framework/pull/26265/commits/efad7e0d5b122fa79c1343ff5163b8149f7796fb